server/inbox/views.py

```diff
@@ -80,6 +83,31 @@ def post(self, request, *args, **kwargs):
             except IntegrityError:
                 return Response({'data':f'You have already sent a like to {object_type} {id_url} on {host_name}.'},
                             status=status.HTTP_200_OK)
+
+            # Sending a LIKE from (us or remote server) to us
+            if (object_host == utils.HOST):
+                try:
+                    Inbox.objects.get(author=request_author_id).send_to_inbox(request.data)
+                except Inbox.DoesNotExist as e:
+                    return Response({'error':'Author not found! Please check author_id in URL.'},
+                                    status=status.HTTP_404_NOT_FOUND)
+            # Sending a LIKE from us to remote server
+            else:
+                try:
+                    remote_server = Node.objects.get(remote_server_url=object_host)
+                except Node.DoesNotExist:
+                    return Response({'error':'Could not find remote server user'}, status=status.HTTP_404_NOT_FOUND)
+
+                headers = {'content-type': 'application/json'}
+                r = requests.post(
+                    f"{object_host}api/author/{request_author_id}/inbox/",
+                    data=json.dumps(request.data),
```

Comment:
you can just use json=json_data here for POSTing JSON, headers will be automatically added

Reply:
ohhhhhh, nice nice. Thanks, Kean, I'll change that now.